crates/sargon/src/profile/v100/address/legacy_olympia_account_address.rs

```diff
@@ -20,10 +19,11 @@ use crate::prelude::*;
 )]
 #[display("{}", self.bech32_address())]
 #[debug("{}", self.bech32_address())]
-pub struct LegacyOlympiaAccountAddressSecretMagic {
-    public_key: Secp256k1PublicKey,
+pub struct LegacyOlympiaAccountAddress {
+    pub value: Secp256k1PublicKey,
```

Comment:
I think it makes sense to keep the name `public_key` to indicate that this address can be parsed from and into a public_key.